app/bingey-ui/src/admin/Dashboard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Header from './components/Navigation/AppBar';
+import UsersList from './components/UsersList';
+import TitleAdd from './components/TitleAdd';
+import UserInfo from './components/UserInfo';
```

Comment:
These should be absolute named imports. Also segregating node-module imports from our own would be better for clarity